src/pages/docs/how-to/http-bridge.mdx

```diff
@@ -0,0 +1,392 @@
+# Use the HTTP API Bridge
+
+Our API is built around [UCAN Invocation](https://github.com/ucan-wg/invocation). This
+allows clients to verify that our service executed a submitted UCAN invocation and produced a
+given result. This gives us a flexible, interoperable, verifiable computational abstraction
+to build upon, but is not widely supported by programming languages or operating system utilities, which
+is a significant disadvantage compared to a more traditional auth-token-based HTTP API.
+
+To compensate for this, we've implemented a "bridge" that lets you use the tools you
+know and love to interact with our API in a manner that should feel familiar if you've
+used an HTTP-based API before.
+
+## Limitations
+
+The HTTP Bridge is currently designed to be used to invoke capabilities on
+spaces. `store/add`, `upload/add` and `upload/list` are examples of such capabilities. Other
```

Comment:
I did not realise we had restricted...perhaps we need a complete list of allowed caps?

Reply:
Yea I'd like to come back to this - the real issue is that the auth tokens currently only allow access to spaces - we'll need a slightly different UCAN to allow invocations against the "service" (ie, `aud: did:web:web3.storage` rather than `aud: did:key:z6Mkabc123`

There's really nothing about the implementation of the bridge that restricts capabilities, I've mostly written it this way to avoid committing to support for anything other than capabilities that act on spaces for now. 

I'll do a pass today and document which capabilities act against spaces and are therefore supported by this version of the documentation!